test/e2e/flowvisibility_test.go

```diff
@@ -462,6 +497,280 @@ func testHelper(t *testing.T, data *TestData, podAIPs, podBIPs, podCIPs, podDIPs
 			checkRecordsForFlows(t, data, podAIPs.ipv4.String(), svcC.Spec.ClusterIP, isServiceIPv6, false, true, false, false)
 		}
 	})
+
+	t.Run("Grafana", func(t *testing.T) {
+		testCases := []struct {
+			dashboardName string
+			dashboardUid  string
+			queryList     []query
+		}{
+			{
+				dashboardName: "flow_records_dashboard",
+				dashboardUid:  "t1UGX7t7k",
+				queryList: []query{
+					{
+						queryId:      2,
+						expectResult: []string{"perftest-a", "perftest-b", "perftest-c", "perftest-d", "perftest-e"},
+					},
+				},
+			},
+			{
+				dashboardName: "pod_to_pod_dashboard",
+				dashboardUid:  "Yxn0Ghh7k",
+				queryList: []query{
+					{
+						queryId:      0,
+						expectResult: []string{"perftest-a", "perftest-b", "perftest-c", "perftest-d", "perftest-e"},
+					},
+					// query{
+					// 	queryId:      1,
+					// 	expectResult: []string{"perftest-a", "perftest-b", "perftest-c", "perftest-d", "perftest-e"},
+					// },
+					// query{
+					// 	queryId:      2,
+					// 	expectResult: []string{"perftest-a", "perftest-b", "perftest-c", "perftest-d", "perftest-e"},
+					// },
+					// query{
+					// 	queryId:      3,
+					// 	expectResult: []string{"perftest-a", "perftest-b", "perftest-c", "perftest-d", "perftest-e"},
+					// },
+					// query{
+					// 	queryId:      4,
+					// 	expectResult: []string{"perftest-a", "perftest-b", "perftest-d"},
+					// },
+					// query{
+					// 	queryId:      5,
+					// 	expectResult: []string{"perftest-a", "perftest-b", "perftest-d"},
+					// },
+					// query{
+					// 	queryId:      6,
+					// 	expectResult: []string{"perftest-a", "perftest-b", "perftest-c", "perftest-d", "perftest-e"},
+					// },
+					// query{
+					// 	queryId:      7,
+					// 	expectResult: []string{"perftest-a", "perftest-b", "perftest-c", "perftest-d", "perftest-e"},
+					// },
```

Comment:
Are these query accidentally commented?

Reply:
No, they are just placeholder so I commented them. I was hoping we can first review whether the test format looks good. Then I will proceed on adding the exact `expectResult` for every single query, given the query amount is very large...